src/si/frequency.rs

```diff
@@ -35,6 +35,12 @@ quantity! {
         @attohertz: prefix!(atto); "aHz", "attohertz", "attohertz";
         @zeptohertz: prefix!(zepto); "zHz", "zeptohertz", "zeptohertz";
         @yoctohertz: prefix!(yocto); "yHz", "yoctohertz", "yoctohertz";
+
+        @cycles_per_shake: 1.0_E-8; "100 MHz", "cycle per shake", "cycles per shake";
+        @cycles_per_minute: 6.0_E1; "1/min", "cycle per minute", "cycles per minute";
+        @cycles_per_hour: 3.6_E3; "1/h", "cycle per hour", "cycles per hour";
+        @cycles_per_day: 8.64_E4; "1/d", "cycle per day", "cycles per day";
+        @cycles_per_year: 3.1536_E7; "1/a", "cycle per year", "cycles per year";
```

Comment:
I went ahead and used `a` as the abbreviation for the year here, following [your lead from `time.rs`](https://github.com/iliekturtles/uom/blob/1d50e8fcc6100876306ef2eda63d1424772e3275/src/si/time.rs#L44).

Reply:
I believe these conversion factors should be inverted. e.g. `1 cycle per minute = 1.0 / 6.0_E1 Hz`. Please double check my math however. If this is the case tests can also be added for `Time` <-> `Frequency`.

Reply:
…I knew something about that seemed off, but I kept convincing myself that "60 Hz is 1 cycle per minute, it's fine." I'll go fix that.